initrd/etc/functions

```diff
@@ -81,6 +81,36 @@ confirm_totp()
 	echo
 }
 
+reseal_tpm_disk_decryption_key()
```

Comment:
For robustness here, I'd suggest also checking `[ "$CONFIG_TPM_NO_LUKS_DISK_UNLOCK" != "y" ]` early in this function and exiting if the feature is disabled.

While there shouldn't be either of those kexec_key_* files in /boot in this case, we would not want users on such a firmware to get into this flow unexpectedly if those files were hanging around somehow.

Reply:
Added in 8809588a :+1: